extensions/cornerstone/src/commandsModule.ts

```diff
@@ -604,6 +609,36 @@ function commandsModule({
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+    setDisplaySetToViewport: ({ displaySet }) => {
```

Comment:
it seems like it is setting on all viewports, so how about `setDisplaySetToGridViewports` and add a comment above explaining what is happening